gramps/gen/datehandler/_date_nl.py

```diff
@@ -83,35 +83,37 @@ class DateParserNL(DateParser):
     month_to_int["xbris"] = 12
 
     modifier_to_int = {
-        'voor'    : Date.MOD_BEFORE,
-        'na'      : Date.MOD_AFTER,
-        'tegen'   : Date.MOD_ABOUT,
-        'om'      : Date.MOD_ABOUT,
-        'rond'    : Date.MOD_ABOUT,
-        'circa'   : Date.MOD_ABOUT,
-        'ca.'     : Date.MOD_ABOUT,
+        'voor'       : Date.MOD_BEFORE,
+        'na'         : Date.MOD_AFTER,
+        'ca.'        : Date.MOD_ABOUT,
+        'circa'      : Date.MOD_ABOUT,
+        'om'         : Date.MOD_ABOUT,
+        'omstreeks'  : Date.MOD_ABOUT,
+        'ongeveer'   : Date.MOD_ABOUT,
+        'rond'       : Date.MOD_ABOUT,
+        'tegen'      : Date.MOD_ABOUT,
         }
 
     calendar_to_int = {
-        'gregoriaans'    : Date.CAL_GREGORIAN,
-        'greg.'          : Date.CAL_GREGORIAN,
-        'juliaans'       : Date.CAL_JULIAN,
-        'jul.'           : Date.CAL_JULIAN,
-        'hebreeuws'      : Date.CAL_HEBREW,
-        'hebr.'          : Date.CAL_HEBREW,
-        'islamitisch'      : Date.CAL_ISLAMIC,
-        'isl.'           : Date.CAL_ISLAMIC,
-        'franse republiek': Date.CAL_FRENCH,
-        'fran.'         : Date.CAL_FRENCH,
-        'persisch'       : Date.CAL_PERSIAN,
-        'zweeds'          : Date.CAL_SWEDISH,
-        'z'               : Date.CAL_SWEDISH,
+        'gregoriaans'         : Date.CAL_GREGORIAN,
+        'greg.'               : Date.CAL_GREGORIAN,
+        'juliaans'            : Date.CAL_JULIAN,
+        'jul.'                : Date.CAL_JULIAN,
+        'hebreeuws'           : Date.CAL_HEBREW,
+        'hebr.'               : Date.CAL_HEBREW,
+        'islamitisch'         : Date.CAL_ISLAMIC,
+        'isl.'                : Date.CAL_ISLAMIC,
+        'frans republiekeins' : Date.CAL_FRENCH,
```

Comment:
This is the correct label, although it, along with the other calendar names, should be capitalized. However, the fact this is a conversion map requires me to ask: does this change break backwards compatibility?

Reply:
As far as I can oversee this only changes the way it is translated into Dutch. This change should not change the functionality.
I have this datehandler running now for a few months and don't experience difficulties yet.